packages/rancher-istio/charts/templates/istio-install-job.yaml

```diff
@@ -41,6 +55,11 @@ spec:
           configMap:
             name: istio-installer-overlay
         {{- end }}
+        {{- if .Values.installer.releaseMirror.provideAdditionalIstioTag }}
+        - name: istio-additional-tag-volume
+          hostPath:
```

Comment:
hostPath is not a good idea for a few reasons, one is that the tar won't exist on every node and we don't yet have nodeSelectors. Lets drop the volume part for now and release not that this only supports upgrading from versions available from rancher 2.5.0-2.5.9

Reply:
https://github.com/rancher/rancher/issues/34858 I have opened an issue here to cover this scenario and I am removing the hostPath change from the chart.